saleor/tests/fixtures.py

```diff
@@ -773,6 +773,23 @@ def color_attribute(db):  # pylint: disable=W0613
     return attribute
 
 
+@pytest.fixture
+def text_attribute(db):
+    attribute = Attribute.objects.create(
+        slug="text",
+        name="Text",
+        type=AttributeInputType.TEXT,
```

Comment:
Still `type` is incorrect, also there is a lack of a test for creating `ProductType` with `text_attribute`.